Source/UI/NewUI/PaymentForm/ViewModel/DefaultPaymentViewModel.swift

```diff
@@ -37,13 +38,40 @@ class DefaultPaymentViewModel: PaymentViewModel {
         updateCardNumberView?()
     }
 
+    func updateBillingSummaryView() {
+        guard let data = billingFormData, paymentFormStyle?.editBillingSummary != nil
+        else { return }
+
+        var summaryValue = ""
+
+        // user name
+        updateSummaryValue(with: data.name, summaryValue: &summaryValue)
+
+        // address
+        if let address = billingFormData?.address {
+            updateSummaryValue(with: address.addressLine1, summaryValue: &summaryValue)
+            updateSummaryValue(with: address.addressLine2, summaryValue: &summaryValue)
+            updateSummaryValue(with: address.city, summaryValue: &summaryValue)
+            updateSummaryValue(with: address.state, summaryValue: &summaryValue)
+            updateSummaryValue(with: address.zip, summaryValue: &summaryValue)
+            updateSummaryValue(with: address.country?.name, summaryValue: &summaryValue)
+        }
+
+        // phone
+        if let phone = billingFormData?.phone {
+            updateSummaryValue(with: phone.number, summaryValue: &summaryValue, withNewLine: false)
+        }
```

Comment:
would it make sense to make updateSummaryValue take an array to avoid so many calls

Reply:
It's already updated in another PR. it's depending now Cell style array